### PR TITLE
Docs work

### PR DESCRIPTION
**Pull request by manolisliolios**

### CODE REVIEW

File: documentation/pages/move-registry.mdx
Comment (stefan-mysten) [inline documentation/pages/move-registry.mdx:3]:
```suggestion
Move Registry provides a name service for interacting and building with packages from the ecosystem.
```

File: documentation/pages/move-registry.mdx
Comment (stefan-mysten) [inline documentation/pages/move-registry.mdx:10]:
```suggestion
## Why use the Move Registry?
```

File: documentation/pages/move-registry.mdx
Comment (stefan-mysten) [inline documentation/pages/move-registry.mdx:12]: Probably this should be in introduction as well.

File: documentation/pages/move-registry.mdx
Comment (stefan-mysten) [inline documentation/pages/move-registry.mdx:13]:
```suggestion
It allows to reference packages by their names, and the Move Registry will resolve the package address, despite the network.
```

File: documentation/pages/move-registry.mdx
Comment (stefan-mysten) [inline documentation/pages/move-registry.mdx:16]:
```suggestion
Additionally, Move Registry helps with working with versioning of packages. Call a specific version of a package easily, without having to 
resolve the addresses or worry about the package being updated. For example, name without a specified version defaults to the latest.
```

File: documentation/pages/move-registry.mdx
Comment (stefan-mysten) [inline documentation/pages/move-registry.mdx:19]:
```suggestion
*Move Registry default service currently supports mainnet and testnet neworks.*
```

File: documentation/pages/move-registry/design.mdx
Comment (stefan-mysten) [inline documentation/pages/move-registry/design.mdx:3]:
```suggestion
Before diving into specifics, let's understand its design. The Move Registry consists of two independent components:
```

File: documentation/pages/move-registry/maintainer-practises.mdx
Comment (stefan-mysten) [inline documentation/pages/move-registry/maintainer-practises.mdx:1]:
```suggestion
# Best Practices
```

File: documentation/pages/move-registry/maintainer-practises.mdx
Comment (stefan-mysten) [inline documentation/pages/move-registry/maintainer-practises.mdx:3]:
```suggestion
For package authors/maintainers, the following best practices are recommended when saving your source code to make sure
```

What do we mean by saving. your source code?

File: documentation/pages/move-registry/maintainer-practises.mdx
Comment (stefan-mysten) [inline documentation/pages/move-registry/maintainer-practises.mdx:8]: is this in Move.toml?

File: documentation/pages/move-registry/maintainer-practises.mdx
Comment (stefan-mysten) [inline documentation/pages/move-registry/maintainer-practises.mdx:1]: I think this would really benefit from an example.

File: documentation/pages/move-registry/tooling/mvr-cli.mdx
Comment (stefan-mysten) [inline documentation/pages/move-registry/tooling/mvr-cli.mdx:18]: we need to figure out how to do this. The simplest for now is to have it via releases. We can also add a script for downloading and installing it, similar to what Walrus used to have.

Simplest version for us:
Go to https://github.com/mystenLabs/mvr/releases and download the binary corresponding to your architecture and OS. Put the binary on the PATH and then use `mvr --help` to see all commands.

File: documentation/pages/move-registry/tooling/typescript-sdk.mdx
Comment (stefan-mysten) [inline documentation/pages/move-registry/tooling/typescript-sdk.mdx:6]: Sorry about my noob question but what is an SDK plugin?

File: documentation/pages/move-registry/tooling/typescript-sdk.mdx
Comment (stefan-mysten) [inline documentation/pages/move-registry/tooling/typescript-sdk.mdx:9]:
```suggestion
This also applies to type names, especially across package upgrades.
```

File: documentation/pages/move-registry/tooling/typescript-sdk.mdx
Comment (stefan-mysten) [inline documentation/pages/move-registry/tooling/typescript-sdk.mdx:13]:
```suggestion
The MVR plugin is exported directly from the @mysten/sui package. To enable the plugin, register it either globally or per PTB in your dApp.
```

File: documentation/pages/move-registry/tooling/typescript-sdk.mdx
Comment (manolisliolios) [reply documentation/pages/move-registry/tooling/typescript-sdk.mdx:6]: Check more info here: 
https://sdk.mystenlabs.com/typescript/plugins

File: documentation/pages/move-registry/tooling/mvr-cli.mdx
Comment (stefan-mysten) [inline documentation/pages/move-registry/tooling/mvr-cli.mdx:18]:
```suggestion
Download the correct binary file according to your OS and architecture from GitHub release page:
https://github.com/MystenLabs/mvr/releases/latest

After downloading the file, put it on your PATH. Below are examples on how to do it on three popular shells:

### For Zsh
1. Open the `zshrc` file
```bash
nano ~/.zshrc
```
2. Add this line at the end, replacing `/path/to/your/folder` with the actual folder path to the binary:
```bash
 export PATH="/path/to/your/folder:$PATH"
```
3. Save the file and source it to apply changes:
```bash
source ~/.zshrc
```

### For bash
1. Open your `.bashrc` file (or `.bash_profile` on macOS):
```bash
nano ~/.bashrc
```
2. Add this line at the end, replacing `/path/to/your/folder` with the actual folder path to the binary:
```bash
 export PATH="/path/to/your/folder:$PATH"
```
3. Save the file and source it to apply changes:
```bash
source ~/.bashrc
```

### For fish
1. Open your Fish configuration file:
```bash
nano ~/.config/fish/config.fish
```
2. Add this line, replacing `/path/to/your/folder` with your desired path:
```bash
set -Ux fish_user_paths /path/to/your/folder $fish_user_paths
```
3. Fish automatically loads this file, so you don’t need to manually source it.

Now you can call `mvr --help`.
```

File: documentation/pages/move-registry.mdx
Comment (tedks) [reply documentation/pages/move-registry.mdx:13]: While we're here could we change "despite" to "regardless of"?

Comment (ronny-mysten) [reply documentation/pages/move-registry.mdx:13]: How's about "irregardless"...

File: documentation/pages/move-registry/tooling/mvr-cli.mdx
Comment (stefan-mysten) [inline documentation/pages/move-registry/tooling/mvr-cli.mdx:17]:
```suggestion

```

**Review by stefan-mysten — APPROVED** [review]
LGTM and quickly iterate once we land it to fix any inconsistencies we might spot afterwards.

File: documentation/pages/move-registry/design.mdx
Comment (tedks) [inline documentation/pages/move-registry/design.mdx:45]: Could we say more about what "single source of truth" means? Such as, "The single source of truth for MVR exists on Mainnet. This means that..."

File: documentation/pages/move-registry/maintainer-practices.mdx
Comment (tedks) [inline documentation/pages/move-registry/maintainer-practices.mdx:11]: Can we link to the AAM concept page here as well?

File: documentation/pages/move-registry/tooling/mvr-cli.mdx
Comment (tedks) [inline documentation/pages/move-registry/tooling/mvr-cli.mdx:33]: I think we could leave this out. There is plenty of documentation on adding binaries to your PATH on various architectures.

**Review by tedks — COMMENTED** [review]
Overall LGTM!

File: documentation/pages/move-registry/maintainer-practices.mdx
Comment (ronny-mysten) [reply documentation/pages/move-registry/maintainer-practices.mdx:11]: The link that follows is for the whole section. Used its own sentence instead of just the title so can tell reader they are opening another site.

File: documentation/pages/move-registry/tooling/mvr-cli.mdx
Comment (ronny-mysten) [reply documentation/pages/move-registry/tooling/mvr-cli.mdx:33]: Leaving for now. It's more of a tool to jog memory than detailed instruction. It's also easy enough for reader to skip over if not needed. Having said that, ask me again and I'll remove because I'm not married to it.

**Comment by ronny-mysten** [comment]
> @manolisliolios @ronny-mysten can we "include" a markdown file here as a section? That might make it easier to keep the installation instructions in the github repository in a markdown file, and just include them here. Alternatively, the repository with the CLI should provide a link to these docs.

This is not a built-in feature, but we can definitely add. I'd prefer the include approach over linking. Linking isn't horrible, but prefer not to make reader jump all over the web.

**Pull request merged**
